src/configurationProvider.ts

```diff
@@ -128,7 +129,7 @@ export class JavaDebugConfigurationProvider implements vscode.DebugConfiguration
         try {
             if (this.isUserSettingsDirty) {
```

Comment:
Could you please help to change the older mechanism of detecting setting dirty of using vscode.workspace.onDidChangeConfiguration and  affectsConfiguration(see debugCodeLensProvider.ts)
instead of isUserSettingsDirty

DebugCodeLensContainer

Reply:
Ok, I’ll take a look

Reply:
Please also update the branch too.